### PR TITLE
Alignment issue resolved

### PR DESCRIPTION
**Pull request by Ayush5071**
### Resolved alignment issue 

https://github.com/user-attachments/assets/d8fa62f8-f902-49e2-900b-5cdd9fb93bf2

### CODE REVIEW

File: client/src/Components/EventsPodium.tsx [hunk 1]
```diff
@@ -22,245 +20,205 @@ export default function EventsPodium({ teams }: Prop) {
...
-                                                            <span></span>
-                                                        </div>
-                                                        <div className='image-cropper'>
-                                                            <img src={`${thirdTeam.members[1].avatar}`} alt="" />
-                                                        </div>
+                    thirdTeam ? (
+                        <div className='events-podium-container' style={{ display: "flex", flexDirection: "column" }}>
+                            <div className="container-events-podium container podium" style={{ transform: "translateY(50px)" }}>
+                                {
+                                    (thirdTeam.members.length >= 2) ? (
```
Comment (meisabhishekpatel) [inline client/src/Components/EventsPodium.tsx:156]: your condition check should be uniform like every you are checking if the team exists or not and then you are checking for count but here you did not check that

**Comment by meisabhishekpatel** [comment]
@Ayush5071 well the issue is still there you don't know how long of a name the winner will have you have to take care of that.

**Comment by Ayush5071** [comment]
Screenshot 2024-10-11 132127
Screenshot 2024-10-11 132147

### I have added Random text to it to make the name long it is working fine ,I have attached the screenShot

**Comment by Ayush5071** [comment]
same PR code can handle long names also , if still i'm missing something please let me know.

**Comment by Ayush5071** [comment]
@meisabhishekpatel sir, can you please look into it

**Comment by meisabhishekpatel** [comment]
@Ayush5071 sorry for late response I was not recieving any email until now,
image


can you see that 1st team's priyanshu's avatar is not perfectly aligned
like the 2nd team, same problem with 3rd team kshatriya's avatar they should be perfectly aligned like the 2nd team where the names are not too big

**Comment by meisabhishekpatel** [comment]
@Ayush5071 you are working on this or not ??

**Comment by Ayush5071** [comment]
@meisabhishekpatel Can i still work ? i thought time is up for the issue. Sir, curently im travelling can't do right now i'll be free tomorrow. Sorry for inconvenience caused sir .

**Comment by meisabhishekpatel** [comment]
@Ayush5071 you can still work, also the timer is only for you to raise a PR.

**Comment by meisabhishekpatel** [comment]
@Ayush5071 you still working on it or not

**Pull request closed without merging**